README.md

```diff
@@ -1,28 +1,157 @@
 # Jaeger OpenShift Templates
 
-Support for deploying Jaeger into OpenShift.
+## Development setup
 
-## Install OpenShift
-If you are new to OpenShift these links help you to get it running:
-* https://install.openshift.com/
-* https://github.com/minishift/minishift
-
-## All-in-one template
-Template with in-memory storage with a limited functionality for local testing and development. 
+This template uses an in-memory storage with a limited functionality for local testing and development.
 Do not use this template in production environments.
 
 To directly install everything you need:
 ```bash
-oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/<version>/all-in-one/jaeger-all-in-one-template.yml | oc create -f -
-oc delete all,template -l jaeger-infra    # to remove everything
+oc new-project jaeger-infra
+oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/master/all-in-one/jaeger-all-in-one-template.yml | oc create -f -
 ```
 
-## Troubleshooting
-Tracer sometimes fails to resolve Jaeger's address. In such a case run:
+Once everything is ready, the Jaeger Query UI can be accessed via https://jaeger-query-jaeger.127.0.0.1.nip.io/ .
+
+## Production setup
+
+### TL;DR
+
+Install with:
 ```bash
-sudo iptables -F
+oc login -u system:admin
+oc new-project jaeger-infra
+oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/master/production/jaeger-production-template.yml | oc create -f -
+oc policy add-role-to-user cluster-admin developer -n jaeger-infra
+```
+
+Once everything is ready, the Jaeger Query UI can be accessed via https://jaeger-query-jaeger.127.0.0.1.nip.io/ .
+
+Your Agent hostname is `jaeger-agent.jaeger-infra.svc.cluster.local`
```

Comment:
Ping works:
```
oc exec -it cassandra-0 bash
bash-4.2$ ping jaeger-agent.jaeger-infra.svc.cluster.local
PING jaeger-agent.jaeger-infra.svc.cluster.local (172.17.0.4) 56(84) bytes of data.
64 bytes from 172.17.0.4 (172.17.0.4): icmp_seq=1 ttl=64 time=0.181 ms
64 bytes from 172.17.0.4 (172.17.0.4): icmp_seq=2 ttl=64 time=0.919 ms
```

It can be verified by using an application that reports spans to the agent, like this:
https://github.com/jpkrohling/hawkular-apm/tree/JPK-JaegerProduction